File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        neg_count = 0
        for num in nums:
            if num == 0:
                return 0
            elif num < 0:
                neg_count+=1
        return 1 if neg_count%2==0 else -1

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        int neg_counter = 0;
        for(int num : nums){
            if(num==0)
                return 0;
            else if(num <0)
                neg_counter++;
        }
        return neg_counter%2==0 ? 1 : -1;
    }
}